src/test/java/suite/optout/TestData.java

```diff
@@ -90,6 +80,59 @@ public static Set<Arguments> optoutTokenPhoneArgs() {
         return args;
     }
 
+    public static Set<Arguments> identityMapEmailArgs() {
+        Set<Operator> operators = getPublicOperators();
+        Set<List<String>> inputs = generateEmailSet(4);
+
+        Set<Arguments> args = new HashSet<>();
+        for (Operator operator : operators) {
+            for (List<String> input : inputs) {
+                args.add(Arguments.of(input.get(0), operator, operator.getName(), input.get(1), input.get(2), Boolean.parseBoolean(input.get(3))));
+            }
+        }
+        return args;
+    }
+
+    public static Set<Arguments> identityMapPhoneArgs() {
+        Set<Operator> operators = getPublicOperators();
+        Set<List<String>> inputs = generatePhoneSet(4);
+
+        Set<Arguments> args = new HashSet<>();
+        for (Operator operator : operators) {
+            for (List<String> input : inputs) {
+                args.add(Arguments.of(input.get(0), operator, operator.getName(), input.get(1), input.get(2), Boolean.parseBoolean(input.get(3))));
+            }
+        }
+        return args;
+    }
+
+    private static Set<List<String>> generateEmailSet(int count) {
```

Comment:
You can update the `optoutTokenEmailArgs()` method in `optout/TestData.java` and just generify it to have both no args and args versions

Same with phone set below